content/legal/_index.md

```diff
@@ -0,0 +1,144 @@
+
+The User is invited to consult them regularly.
+
+The use of the Website is free.
+
+**1. Purpose of the Website.**
+
+The purpose of the Website is to provide documentation to the PrestaShop project contributors.
+
+The PrestaShop project is a universal open-source software platform to build your e-commerce solution.
+ 
+**2. Intellectual Property.**
+
+All Website elements and their design, selection, presentation and organization are protected by intellectual property rights owned by PrestaShop. 
+
+The trademarks, domain names, acronyms, logos, drawings and other elements that appear on the Website are the exclusive property of PrestaShop.
+
+Users are further informed that using the PrestaShop trademark in a domain name is strictly prohibited.
+
+Consequently, Users agree not to use this trademark in a domain name.
+
+Using the PrestaShop trademark in a domain name may result in legal proceedings.
+
+Using the Website does not transfer any intellectual property rights, including those relating to its content, the PrestaShop trademark or the software.
+
+**3. PrestaShop’s liability and guarantees.**
+
+PrestaShop will do its best to ensure the Website is accessible 24 hours a day and 7 days a week, without however being bound by an obligation to achieve specific results.
+
+However, PrestaShop may suspend, restrict or interrupt access to the Website or certain pages on it at any time in order to carry out updates, content modification and any other action deemed necessary for the Website to operate correctly.
+
+As such, PrestaShop in no way guarantees the availability, access to or continuity of the Website.
+
+Should it become unavailable for any reason, this will not give rise to any compensation for the User.
+
+PrestaShop is only responsible for the content it has published itself.
+
+PrestaShop shall not be held liable for any damage or loss suffered by the User in accessing and using the Website.
+
+More generally, PrestaShop shall not be held liable for:
+
+* The use made by the User of the Website,
+* Any activity that constitutes a breach of these T&Cs by the User,
+* Any damage caused to the User’s equipment by a virus contracted while browsing the Website or any other damage caused by the User’s browsing.
+
+The User is solely liable for all property or consequential financial loss or damage resulting from the use of the Website.
+
+**4. Hyperlinks.**
+
+The Website may contain hyperlinks to redirect User to third party websites, including to agency or partner websites, or social networks.
+
+PrestaShop accepts no liability for the websites to which these links lead as it has no control over the various third party websites.
+
+Websites linking to pages on our Website may not reproduce its pages in any way.
```

Comment:
This is not compatible with the site's license and must be updated

Reply:
It's not consistent with the content of the page being released under Creative Commons. This is something that the legal people, not you or me, should be writing. Perhaps we should go back to a Google doc for the review process with them.

Reply:
Hello @eternoendless, as suggested by legal, I'm deleting this sentence.